registry/nacos/service_discovery.go

```diff
@@ -237,7 +237,7 @@ func (n *nacosServiceDiscovery) DispatchEventForInstances(serviceName string, in
 
 // DispatchEvent will dispatch the event
 func (n *nacosServiceDiscovery) DispatchEvent(event *registry.ServiceInstancesChangedEvent) error {
-	// TODO(waiting for event dispatcher, another task)
+	extension.GetGlobalDispatcher().Dispatch(event)
```

Comment:
In the other implements.Should they call ```extension.GetGlobalDispatcher().Dispatch(event)``` in `DispatchEvent`  mehod as well?

Reply:
Yes, but I think we could allow users to configure the dispatcher.